controller/setting_controller.go

```diff
@@ -403,6 +428,90 @@ func (sc *SettingController) updatePriorityClass() error {
 	return nil
 }
 
+func (sc *SettingController) updateDefaultStorageClass() error {
+	storageConfigMap, err := sc.ds.GetConfigMapRO(types.DefaultStorageClassConfigMapName)
+	if err != nil {
+		if datastore.ErrorIsNotFound(err) {
+			// The storageclass configmap is deleted therefore we delete the actual storageclass
+			err := sc.ds.DeleteStorageClass(types.DefaultStorageClassName)
+			if err != nil && !datastore.ErrorIsNotFound(err) {
+				return nil
```

Comment:
Better to return the error instead?

Reply:
Yeah. We should return err here. Thank you! Fixed at https://github.com/longhorn/longhorn-manager/pull/738